app/_src/gateway/production/sizing-guidelines.md

```diff
@@ -169,6 +170,17 @@ the better choice, as it provides near-unlimited throughput capacity. In this
 scenario, adding more cache memory would not increase maximum throughput by
 much.
 
+{% if_version gte:3.5.x %}
+In {{site.base_gateway}} setups with high cardinality of configured Routes, Services,
+Plugins, Consumers etc, the cluster may benefit from "Dedicated Config Processing". The
+feature presents an innovative approach to enhance system performance by employing a dedicated
+worker process. By offloading specific tasks, it promises reduced latencies and offers a more
+efficient operational experience, when operating in `hybrid mode``.
+Users should remain aware of the slight increase in memory usage but can benefit significantly
+from the accelerated processing times.
+(add_me: link to "Reference -> kong.conf -> "dedicated config processing")
```

Comment:
@Kong/docs-maintainers  Two questions:

1. Do we update the `References -> kong.conf` section automatically with new config options?

2. How do you link to this section?

Reply:
> Do we update the References -> kong.conf section automatically with new config options?

Pseudo-automatically, but yes. We'll have that generated today.

> How do you link to this section?

The link would be `/gateway/{{page.kong_version}}/reference/configuration/#dedicated_config_processing`.

Reply:
ty, added